ansible-ipi-install/inventory/jetski/hosts

```diff
@@ -43,10 +43,14 @@ ipv6_enabled=False
 
 # (Optional) When ipv6_enabled is set to True, but want IPv4 addressing on provisioning network
 # Default is false.
-#ipv4_provisioning=True
+ipv4_provisioning=False
 
 # (Optional) When ipv6_enabled is set to True, but want IPv4 addressing on baremetal network
-#ipv4_baremetal=True
+ipv4_baremetal=False
+
+# (Optional) When ipv6_enabled is set to True, but want dual-stack for baremetal network
+# Only one of ipv4_baremetal and dualstack_baremetal can be true 
```

Comment:
May be good to mention about disabling lab router advertisement for dualstack.

Reply:
Yeh, just want to use the same text as in upstream inventory file. Not sure RA's are a problem for everyone :)